ext/web/06_streams.js

```diff
@@ -654,31 +654,32 @@
    * read directly from the underlying resource if they so choose (FastStream).
    *
    * @param {number} rid The resource ID to read from.
+   * @param {boolean=} autoClose If the resource should be auto-closed when the stream closes. Defaults to true.
```

Comment:
JSDoc would also support defining default value like `@param {boolean=true}`. I wonder if denoland would survive that?

Reply:
Nope, typescript doesn't seem to like it.